app/controllers/admin/edition_translations_controller.rb

```diff
@@ -7,9 +7,18 @@ def new
     render "legacy_new" unless preview_design_system_user?
   end
 
+  def edit
+    render "edit_legacy" unless preview_design_system_user?
+  end
+
   def update
     @translated_edition.change_note = "Added translation" if @translated_edition.change_note.blank?
-    super
+    if translatable_item.update(translation_params)
+      save_draft_translation if send_downstream?
+      redirect_to update_redirect_path, notice: notice_message("saved")
```

Comment:
@dnkrj could you add some context regarding this change to the commit message? Why it's needed etc.